main.js

```diff
@@ -0,0 +1,213 @@
+// 1.
+/*Створити функцію-конструктор Людини.
+ Властивості, які будуть описувати екземпляр Людини - ім'я, вік, стать, національність, країна, список країн для подорожей.
+ Створити універсальні функції - знайомство, прокидатися, засинати й список країн,
+ які Людина хоче відвідати.Роботу цих функцій можна виводити через console.log.Але
+ в кожній з функцій повинно використовуватись якомога більше
+ характеристик конкретної Людини на якій ця функція викликається(підказка: тут треба використатии call / apply / bind) */
+
+function CreateHumanList(name, age, sex, nationality, country, []) {
+  this.nameOfHuman = name;
+  this.ageOfHuman = age;
+  this.sexOfHuman = sex;
+  this.nationalityOfHuman = nationality;
+  this.countryOfHuman = country;
+  this.countryToTravel = [];
+}
+
+const humanMe = new CreateHumanList(
+  "Vladyslav",
+  34,
+  "Male",
+  "Ukrainian",
+  "Ukraine",
+  ["Italy", "Spain", "Poland", "Portugal"]
+);
+
+const greet = function () {
+  console.log(`Hello ${this.nameOfHuman} have a nice day!`);
+};
+greet.apply(humanMe);
+
+const wakeUp = function () {
+  console.log(`Wake up ${this.nameOfHuman} already, you're late for work!`);
+};
+wakeUp.apply(humanMe);
+
+const sleep = function () {
+  console.log(
+    `I'm very tired today, I'm going to sleep and dream about ${["Portugal"]}.`
+  );
+};
+sleep.apply(humanMe);
+
+const travel = function () {
+  console.log(
+    `At my ${this.ageOfHuman} years old, I plan to go ${[
+      "Poland",
```

Comment:
Assuming this code - every human would love to travel to Poland, which is not according to task

At least try log the first country from array:

```suggestion
    `At my ${this.ageOfHuman} years old, I plan to go ${this.listOfCountries[0],
```